<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tw_livros">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>59</y>
      <width>471</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Inserir Livro</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>271</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="lb_categ">
         <property name="text">
          <string>Categoria*</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_nome"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_isbn"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_isbn">
         <property name="text">
          <string>ISBN*</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_categ"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_autor">
         <property name="text">
          <string>Autor*</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_nome">
         <property name="text">
          <string>Nome*</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_autor"/>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="bt_enviar">
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab4">
     <attribute name="title">
      <string>Editar Livro</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>271</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="lb_categ_2">
         <property name="text">
          <string>Categoria*</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_nome_2"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_isbn_2"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_isbn_2">
         <property name="text">
          <string>ISBN*</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_categ_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_autor_2">
         <property name="text">
          <string>Autor*</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_nome_2">
         <property name="text">
          <string>Nome*</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_autor_2"/>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="bt_atualizar">
         <property name="text">
          <string>Atualizar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="cb_pesq">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ISBN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nome</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Categoria</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="le_pesq">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>183</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_pesq">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Pesquisar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>Remover Livro</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>451</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_remov"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ISBN</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="bt_remov">
         <property name="text">
          <string>Remover</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_biblioteca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Biblioteca</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
